CPackConfig.txt

```diff
@@ -0,0 +1,79 @@
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Lightweight multi-architecture disassembly framework - devel files")
+set(CPACK_PACKAGE_HOMEPAGE_URL "https://www.capstone-engine.org/")
+set(CPACK_STRIP_FILES false)
+
+# Set Debian-specific package variables
+set(CPACK_DEBIAN_PACKAGE_NAME "libcapstone-dev")
+set(CPACK_DEBIAN_PACKAGE_SOURCE "capstone")
+set(CPACK_DEBIAN_PACKAGE_VERSION "${PROJECT_VERSION}")
+set(CPACK_DEBIAN_PACKAGE_ORIGINAL_MAINTAINER "Debian Security Tools <team+pkg-security@tracker.debian.org>")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "libc6 (>= 2.2.5)")
+set(CPACK_DEBIAN_PACKAGE_SECTION "libdevel")
+set(CPACK_DEBIAN_PACKAGE_PRIORITY "optional")
+set(CPACK_DEBIAN_PACKAGE_MULTIARCH "same")
+
+# Determine architecture
+if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64")
+    set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "amd64")
+elseif(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "i386" OR ${CMAKE_SYSTEM_PROCESSOR} STREQUAL "i686")
+    set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "i386")
+elseif(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "arm")
+    if(CMAKE_SIZE_OF_VOID_P EQUAL 4)
+        set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "armhf")
+    else()
+        set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "arm64")
+    endif()
+else()
+    set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE ${CMAKE_SYSTEM_PROCESSOR})
+endif()
+
+# Copy over Debian documentation
+install(FILES 
+    "${CMAKE_SOURCE_DIR}/ChangeLog" 
+    "${CMAKE_SOURCE_DIR}/CREDITS.TXT" 
+    "${CMAKE_SOURCE_DIR}/README.md" 
+    "${CMAKE_SOURCE_DIR}/SPONSORS.TXT"
+    DESTINATION "/usr/share/doc/libcapstone-dev"
+)
+install(DIRECTORY 
+    "${CMAKE_SOURCE_DIR}/LICENSES"
+    DESTINATION "/usr/share/doc/libcapstone-dev"
+)
+message(STATUS "Adding documentation files for DEB generator")
+
+# Include additional file to run 'ldconfig' after install
+set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA "${CMAKE_SOURCE_DIR}/packages/deb/triggers")
+
+# RPM package settings
+set(CPACK_RPM_PACKAGE_NAME "capstone-devel")
+set(CPACK_RPM_PACKAGE_VERSION "${PROJECT_VERSION}")
+# set(CPACK_RPM_PACKAGE_RELEASE "6.fc42")
+set(CPACK_RPM_PACKAGE_ARCHITECTURE ${CMAKE_SYSTEM_PROCESSOR})
+set(CPACK_RPM_PACKAGE_GROUP "Development/Libraries")
+set(CPACK_RPM_PACKAGE_REQUIRES "libc6 >= 2.2.5")
+set(CPACK_RPM_POST_INSTALL_SCRIPT_FILE "${CMAKE_SOURCE_DIR}/packages/rpm/postinstall.sh")
+set(CPACK_RPM_POST_UNINSTALL_SCRIPT_FILE "${CMAKE_SOURCE_DIR}/packages/rpm/postinstall.sh")
+set(CPACK_RPM_CHANGELOG_FILE "${CMAKE_SOURCE_DIR}/ChangeLog")
+set(CPACK_RPM_PACKAGE_LICENSE "BSD3, LLVM")
+set(CPACK_RPM_PACKAGE_DESCRIPTION "${CPACK_PACKAGE_DESCRIPTION}")
+
+# TODO: Do you have a script to run tests after installing the RPM/Debian package?
+# Add %check section to RPM spec file
+# set(CPACK_RPM_SPEC_MORE_DEFINE "${CPACK_RPM_SPEC_MORE_DEFINE}\n%check\ntrue")
+
+# TODO: Please help get CPack working for Mac OSX packages too
+# OSX package settings
+# set(CPACK_BUNDLE_NAME "Capstone")
+# set(CPACK_BUNDLE_PLIST "xcode/CapstoneFramework/Info.plist")
```

Comment:
Remove unused code. Please open an issue about it when this is merged. Although, IMHO, For the CPack generated packages we don't need desperately tests. I think it is reasonable to trust the CPack folks.